security/yubi-key.md

```diff
@@ -23,27 +23,27 @@ This can be configured using
 package. This package does not support sharing the same key slot with other
 applications (it will deny further authentications if you try).
 
-Contrary to instruction there, currently there is no binary packages in Qubes
+Contrary to instruction there, currently there is no binary package in the Qubes
 repository and you need to compile it yourself. This might change in the future.
```

Comment:
AFAICT has changed. We now have `qubes-yubikey-dom0`.

Either way, IMO that should be addressed in a separate PR.